### PR TITLE
feat(ui): add broadcasts display in UI

### PR DESCRIPTION
**Pull request by bnjjj**
1. Description
1. Related issues
1. About tests
unit tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: engine/api/broadcast/broadcast.go [hunk 1]
```diff
@@ -29,46 +29,88 @@ func Update(db gorp.SqlExecutor, bc *sdk.Broadcast) error {
 	return nil
 }
 
+// MarkAsRead mark the broadcast as read for an user
```
Comment (sguiheux) [inline engine/api/broadcast/broadcast.go]: gorp .Insert ?

File: engine/api/broadcast/broadcast.go [hunk 2]
```diff
@@ -29,46 +29,88 @@ func Update(db gorp.SqlExecutor, bc *sdk.Broadcast) error {
...
+	return sdk.WrapError(err, "MarkAsRead>")
+}
+
 // LoadByID loads broadcast by id
-func LoadByID(db gorp.SqlExecutor, id int64) (*sdk.Broadcast, error) {
-	dbBroadcast := broadcast{}
-	query := `select * from broadcast where id=$1`
-	if err := db.SelectOne(&dbBroadcast, query, id); err != nil {
+func LoadByID(db gorp.SqlExecutor, id int64, u *sdk.User) (*sdk.Broadcast, error) {
+	var projectKey sql.NullString
```
Comment (sguiheux) [inline engine/api/broadcast/broadcast.go]: use gorp

File: engine/api/broadcast/broadcast.go [hunk 3]
```diff
@@ -29,46 +29,88 @@ func Update(db gorp.SqlExecutor, bc *sdk.Broadcast) error {
...
 
-	broadcasts := make([]sdk.Broadcast, len(res))
-	for i := range res {
-		p := res[i]
-		broadcasts[i] = sdk.Broadcast(p)
+	broadcasts := []sdk.Broadcast{}
+	for rows.Next() {
+		var projectKey sql.NullString
+		var broadcast sdk.Broadcast
+		err := rows.Scan(&broadcast.ID, &broadcast.Title, &broadcast.Content, &broadcast.Level,
```
Comment (sguiheux) [inline engine/api/broadcast/broadcast.go]: one line

File: engine/api/broadcast/broadcast.go [hunk 4]
```diff
@@ -29,46 +29,88 @@ func Update(db gorp.SqlExecutor, bc *sdk.Broadcast) error {
...
+			broadcast.project_id,
+			project.projectkey,
+			(broadcast_read.broadcast_id IS NOT NULL)::boolean AS read
+			FROM broadcast
+				LEFT JOIN broadcast_read ON broadcast.id = broadcast_read.broadcast_id AND broadcast_read.user_id = $1
+				LEFT JOIN project ON broadcast.project_id = project.id
+		WHERE broadcast.id = $2
+	`
+	var broadcast sdk.Broadcast
+	err := db.QueryRow(query, u.ID, id).Scan(&broadcast.ID, &broadcast.Title, &broadcast.Content, &broadcast.Level,
```
Comment (sguiheux) [inline engine/api/broadcast/broadcast.go]: one line

File: engine/api/user.go [hunk 5]
```diff
@@ -256,7 +256,12 @@ func (api *API) addUserHandler() Handler {
 			return sdk.WrapError(err, "AddUser: Cannot insert user")
 		}
 
-		go mail.SendMailVerifyToken(createUserRequest.User.Email, createUserRequest.User.Username, tokenVerify, createUserRequest.Callback)
+		go func() {
+			errM := mail.SendMailVerifyToken(createUserRequest.User.Email, createUserRequest.User.Username, tokenVerify, createUserRequest.Callback)
```
Comment (sguiheux) [inline engine/api/user.go]: one line

File: engine/api/broadcast/broadcast.go [hunk 6]
```diff
@@ -29,46 +29,88 @@ func Update(db gorp.SqlExecutor, bc *sdk.Broadcast) error {
...
 	return &broadcast, nil
 }
 
 // LoadAll retrieves broadcasts from database
-func LoadAll(db gorp.SqlExecutor) ([]sdk.Broadcast, error) {
-	res := []broadcast{}
-	if _, err := db.Select(&res, `select * from broadcast`); err != nil {
-		return nil, sdk.WrapError(err, "LoadAllBroadcasts> ")
+func LoadAll(db gorp.SqlExecutor, u *sdk.User) ([]sdk.Broadcast, error) {
+	query := `
```
Comment (sguiheux) [inline engine/api/broadcast/broadcast.go]: gorp?

File: engine/api/broadcast/broadcast.go (hunk 4 above)
Comment (bnjjj) [reply engine/api/broadcast/broadcast.go]: no, become too unreadable in inline

File: engine/api/broadcast/broadcast.go (hunk 3 above)
Comment (bnjjj) [reply engine/api/broadcast/broadcast.go]: no, become too unreadable in inline

File: engine/api/broadcast/broadcast.go (hunk 1 above)
Comment (bnjjj) [reply engine/api/broadcast/broadcast.go]: done

**Pull request merged**